### ISSUE TITLE
There is no app.test.js file, referenced from README

### ISSUE THREAD

**Issue by n2o** [qunabu/strapi-unit-test-example#62]
Hi,

you are referencing the app.test.js in your README, but this file does no longer exist.

https://github.com/qunabu/strapi-unit-test-example/blob/main/README.md?plain=1#L9

https://github.com/qunabu/strapi-unit-test-example/blob/main/tests/app.test.js

**Comment by qunabu** [comment]
this file was required with strapi 3 because tested needed to be composed in one file 
in s4 all tests are separated now. 

Readme is updated

**Comment by n2o** [comment]
Thanks 👍

**Comment by n2o** [comment]
Thanks for this repo. I tried to implement tests in my strapi application and was really stuck. Following your code was way better than an outdated doc ;-)

I had to remove the `.load()` from the Strapi call in strapi 4.4.1, but I am not sure why. Now it works as expected with `await Strapi();`.

I just started with the tests. So I will figure out why this was not working for me.

Sad, that strapi is lacking a good documentation for using tests...
